baselines/pyproject.toml

```diff
@@ -38,7 +38,7 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.7.12"
+python = "^3.8.16"
```

Comment:
Why is it 3.8.16 here, but 3.8.15 in the GH Actions workflow?

Reply:
Because the GH Actions isn't compatible with `3.8.16` yet, but indeed I might change the baseline version to `3.8.15` for consistency.